packages/contentstack-bulk-publish/src/consumer/publish.js

```diff
@@ -25,13 +24,29 @@ function initializeLogger(fileName) {
 /* eslint-disable camelcase */
 function removePublishDetails(elements) {
   if (elements && elements.length > 0) {
-    return elements.map(({ _publish_details, ...rest }) => rest);
+    elements.map((element) => {
+      if (element.hasOwnProperty('publish_details')) {
+        delete element.publish_details;
+      }
+      return element;
+    });
   } else {
     delete elements.publish_details;
   }
   return elements;
 }
 
+function displayEntriesDetails(sanitizedData) {
+  sanitizedData.forEach((entry) => {
+    console.log(chalk.green(`Entry UID '${entry.uid}' of CT '${entry.content_type}' in locale '${entry.locale}'`));
+  });
+}
+
+function displayAssetsDetails(sanitizedData) {
+  sanitizedData.forEach((asset) => {
+    console.log(chalk.green(`Asset UID '${asset.uid}' ${asset.version ? `and version '${asset.version}'` : ''} ${asset.locale ? `in locale '${asset.locale}'` : ''}`));
+  });
+}
```

Comment:
@cs-raj , formatting missing

Reply:
Added, not sure how only this line it was left